app/crawler/mbc/api.py

```diff
@@ -1,76 +1,113 @@
 import json
-from datetime import date
-from typing import Any
+from datetime import datetime
+from typing import TypedDict
 
 import aiohttp
+from pydantic import BaseModel
 
-from app.crawler.base import Article, BaseNewsCrawler
+from app.crawler.base import BaseNewsCrawler
+from app.crawler.utils.headers import create_mbc_headers
 from app.crawler.utils.json_cleaner import sanitize_js_style_json
+from app.schemas.article import ArticleDTO, ArticleMetadata
 from common.utils.logger import get_logger
 
 logger = get_logger(__name__)
 
 
+class HeadlineArticle(TypedDict):
+    Section: str
+    AId: str
+    Title: str
+    Desc: str
+    Image: str
+    Link: str
+    IsVod: str
```

Comment:
플랫폼이 제공하는 데이터구조때문에 속성이름을 snake_case 로 작성할 수 없는 문제가 있어보이네요

이런 이유로 데이터를 유연하게 다루기 위해 TypedDict 를 상속받고 키값에 접근하면서 후처리 하고계셨을 것이라 생각했습니다.

pydantic.Field 의 alias 를 사용해보실까요?

후처리 하는 부분도 꽤 깔끔해질거에요


```suggestion
from typing import Annotated
from pydantic import BaseModel, Field

class HeadlineArticle(BaseModel):
    section: Annotated[str, Field(description="섹션", alias="Section")]
    article_id: Annotated[str, Field(description="기사 ID", alias="AId")]
    title: Annotated[str, Field(description="기사 제목", alias="Title")]
    description: Annotated[str, Field(description="기사 요약", alias="Desc")]
    image_url: Annotated[str, Field(description="기사 이미지 URL", alias="Image")]
    url: Annotated[str, Field(description="기사 링크", alias="Link")]
    is_vod: Annotated[str, Field(description="비디오 여부", alias="IsVod")]
```